khard/carddav_object.py

```diff
@@ -448,6 +449,11 @@ def _prepare_birthday_value(self, date: Date) -> Tuple[Optional[str],
                 fmt = "%F"
         return date.strftime(fmt), False
 
+    @property
+    def kind(self) -> str:
+        kind = self._get_string_field("kind") or self._default_kind
+        return kind if kind != "org" else "organisation"
```

Comment:
I'm uncertain whether `org` should be displayed as `organisation`…
`khard` requires to use `organisations` when querying for the organisation. Yet the corresponding value of the `KIND` property is defined as `org` (see [RFC 6350 6.1.4 KIND](https://datatracker.ietf.org/doc/html/rfc6350#section-6.1.4)) and the RFC always spells it as `organization` (note the use of `z` instead of `s`).

What are your thoughts on this @lucc

Reply:
Sticking with the British spelling as `khard` already uses "organisation" rather than "organization" seems to a good solution for the time being then.